### PR TITLE
[DO NOT MERGE] PS LP updates

### PR DESCRIPTION
**Pull request by heyawhite**
Changes proposed in this pull request:

- Small design nits
- Japanese landing page (WIP)

### CODE REVIEW

**Comment by heyawhite** [comment]
@agektmr FYI I'm going to start work here, but what I need to figure out next is an i18n import for all of the custom descriptions, instead of relying on total copies.

More work on this next week.

**Comment by agektmr** [comment]
IIUC the preview on the staging server won't include Japanese. I can check it on my local env next week. Thank you, Alexandra!

File: site/ja/privacy-sandbox/index.njk [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  "image/VbsHyyQopiec0718rMq2kTE1hke2/QDyL1Otpv6Id7sDTconF.svg",
+  "More documentation",
+  sections.intro_related
+  )
+}}
+
+{{ landingSectionUseCases(
+  "隠されたトラッキングを防ぐ",
+  "Proposals to prevent third-parties from fingerprinting or collecting unlimited sensitive data.",
+  "User-Agent reduction",
```
Comment (heyawhite) [inline site/ja/privacy-sandbox/index.njk:90]:
```suggestion
  "User-Agent の情報量削減",
```

**Comment by heyawhite** [comment]
@agektmr I am curious if you're able to preview it, because I wasn't. 

There are a couple of things that should probably be localized to make this document more accessible, such as the "Read the documentation." button and the subtitles for each of the use case sections.

**Review by heyawhite — COMMENTED** [review]
One l18n effort, copied from the Japanese docs

**Comment by agektmr** [comment]
I've updated Japanese text as much as possible. And here's the screenshot.

localhost_8080_ja_privacy-sandbox_

BTW, I noticed you omit links to a few of the articles that exist in English but not in Japanese. Is this intentional? I think we can keep them the same with English version on all languages, otherwise the readers will miss out and the maintenance overhead is big. I think it's fine English contents are included in other languages. What do you think?

**Comment by heyawhite** [comment]
Amazing, thank you @agektmr! I actually haven't changed any of think links, I totally agree they are still valuable. My guess is because the two attribution reporting docs are new and haven't yet be localized, they won't populate in the Japanese page.

I believe you opened an Issue about this in April: https://github.com/GoogleChrome/developer.chrome.com/issues/2639

**Review by agektmr — APPROVED** [review]
I see, thanks @heyawhite for the clarification. If this is a separate issue, I think we are good to merge this.

**Pull request merged**